pages/blog/event-driven-api-documentation-made-simple-clientside-rendering.md

```diff
@@ -0,0 +1,219 @@
+
+  config='{"show": {"sidebar": false}}'
+
+  cssImportPath="https://unpkg.com/@asyncapi/react-component@1.0.0-next.39/styles/default.min.css">
+</asyncapi-component>
+```
+
+2️⃣ If you need support for old browsers then you need to add this script as well `<script src="https://unpkg.com/@webcomponents/webcomponentsjs@2.5.0/webcomponents-bundle.js"></script>`.
+
+That is it! 🤯 Just awesome!
+
+---
+
+### <a name="sb"></a>Standalone Bundle
+
+If you want to render documentation from your AsyncAPI file without the use of any framework but with just an HTML webpage then you will need the [Standalone bundle](https://github.com/asyncapi/asyncapi-react/blob/next/docs/usage/standalone-bundle.md).
+
+1️⃣ All you need is just a basic HTML template.
+
+2️⃣ In the **head** element enter `<link rel="stylesheet" href="https://unpkg.com/@asyncapi/react-component@1.0.0-next.39/styles/default.min.css">` to get AsyncAPI document styling.
+
+3️⃣ In the **body** element type in:
+
+```
+ <div id="asyncapi"></div>
+
+    <script src="https://unpkg.com/@asyncapi/react-component@1.0.0-next.39/browser/standalone/index.js"></script>
+    <script>
+        AsyncApiStandalone.render({
+            schema: '{"asyncapi":"2.4.0","info":{"title":"Account Service","version":"1.0.0","description":"This service is in charge of processing user signups"},"channels":{"user/signedup":{"subscribe":{"message":{"$ref":"#/components/messages/UserSignedUp"}}}},"components":{"messages":{"UserSignedUp":{"payload":{"type":"object","properties":{"displayName":{"type":"string","description":"Name of the user"},"email":{"type":"string","format":"email","description":"Email of the user"}}}}}}}'
+            ,
+            config: {
+                show: {
+                    sidebar: false,
+                }
+            },
+        }, document.getElementById('asyncapi'));
+    </script>
+```
+
+`<script src="https://unpkg.com/@asyncapi/react-component@1.0.0-next.39/browser/standalone/index.js"></script>` fetches everything required from the bundle.
+
+I almost forgot.
+
+There is one more way to configure the AsyncAPI component.
+You can do it through **config** props, this same as for normal React component.
+
+My **Web Component** and **Standalone Bundle** usage examples have `config='{"show": {"sidebar": false}}` which turns a sidebar off, but if you change it to _**true**_ then your rendered document will have a sidebar.
+
+This was only a sample of what could be done. AsyncAPI can do a lot more. Check [AsyncAPI documentation](https://www.asyncapi.com/docs/tutorials/getting-started/asyncapi-documents) for more information on how to generate docs.
```

Comment:
I don't think so that link is good - atm we don't have any docs for generating in our website. Maybe you had in mind your blog posts in your site, hmm? :)

Reply:
Also, you can mention at the end (or create separate section for that) that AsyncAPI Standalone Bundle can be used in Angular 2+ apps and points to the official docs https://github.com/asyncapi/asyncapi-react/blob/next/docs/usage/angular.md WDYT?